src/stores/oidc/OidcClientStore.ts

```diff
@@ -25,6 +24,10 @@ import PlatformPeg from "../../PlatformPeg";
 /**
  * @experimental
  * Stores information about configured OIDC provider
+ * In OIDC Native mode the client is registered with OIDC directly and maintains an OIDC token
+ * In OIDC Aware mode, the client is aware that the Server is using OIDC, but is using the standard Matrix APIs for most things,
+ * notable exceptions are account management where a link to the account management endpoint will be provided instead.
+ * Otherwise, the store is not operating, the auth is then in Legacy mode and everything uses normal Matrix APIs.
```

Comment:
Nits.

```suggestion
 *
 * In OIDC Native mode the client is registered with OIDC directly and maintains an OIDC token.
 *
 * In OIDC Aware mode, the client is aware that the Server is using OIDC, but is using the standard Matrix APIs for most things.
 * (Notable exceptions are account management,where a link to the account management endpoint will be provided instead.)
 *
 * Otherwise, the store is not operating. Auth is then in Legacy mode and everything uses normal Matrix APIs.
```